docs/content-editing/index.md

```diff
@@ -9,10 +9,10 @@ myst:
 
 (content-editing-label)=
 
-# Content Editing for Plone
+# Content Editing for Plone Volto
 
 About
-: How to create and manage content using Plone
+: How to create and manage content using the Plone Volto frontend
```

Comment:
I'll go along with more detail here, since it won't appear as a primary navigation item or a title of the training. But let's use appropriate terminology.

```suggestion
: How to create and manage content using Plone with Volto as its frontend
```